docs/Processes.md

```diff
@@ -154,14 +154,18 @@ not completed in the alotted time, and return a timeout error to the
 client. If the query is already running on any backend servers then they
 cannot be timed out other than by using the standard -T flag.
 
+As a default, the gateway is set to only take synchronous calls with 
+`.gw.synccallsallowed`. To allow asynchronous calls, edit the 
+gateway.q file so that .gw.synccallsallowed is set to false.
```

Comment:
Is this still the case? I think if it can detect kdb 3.6 then it should just allow them

Reply:
is this description correct? if so, it seems counter-intuitive to me that `.gw.synccallsallowed` defines whether or not _async_ calls are allowed

Reply:
Am I right in saying if kdb v3.6 is being used, async calls are allowed and if not the user will have to set `.gw.synccallsallowed` to 0b?
Should I remove this paragraph completely or edit it to explain that with kdb versions older than 3.6, the user will potentially need to set .gw.synccallsallowed?

 I'll look into the code now, but when I was running the gateway in debug mode, I couldn't use async calls from a client unless I set `.gw.synccallsallowed` to 0b in the gateway.

Reply:
I'd edit it rather than remove it. Definitely could be that you have to disable sync calls to allow async calls, just seems a little non-obvious to me (but that's more to do with the code itself than this documentation)

Reply:
So 3.6 should allow both at the same time since kx put in that new deferred response mechanism. Pre 3.6 the gateways had to allow one or the other, otherwise life got difficult (e.g. someone could come in with an sync request and just ruin everyone else's good async work)